Automattic/jetpack
# Build: Add run-s to some npm scripts

`run-s` (part of [npm-run-all](https://www.npmjs.com/package/npm-run-all) is a handy tool for running npm scripts in series or parallel.

Observation: This is exploratory work. We have so many non-script tasks defined as scripts in package.json... I'm having second thoughts about if this is actually helpful

#### Changes proposed in this Pull Request:

* Introduces `npm-run-all` as dependency
* Makes the  `yarn build` use `run-s` 

#### Testing instructions:

* Checkout this branch
* Run `yarn distclean`
* Run `yarn build`.
* Confirm it builds the admin page properly.

#### Proposed changelog entry for your changes:

* None needed
